src/js/modules/sections/actions.js

```diff
@@ -36,20 +31,51 @@ export const createSection = section => dispatch => {
     payload: section
   });
   axios
-  .post(`${STUY_SPEC_API_URL}/sections`, section, AXIOS_CONFIG)
-  .then(response => {
-    dispatch({
-      type: t.CREATE_SECTION_SUCCEEDED,
-      payload: response.data
-    });
-    dispatch(fetchSections());
-    dispatch(push("/"));
-  })
-  // TODO: Create error messages for requests
-  .catch(error => {
-    dispatch({
-      type: t.CREATE_SECTION_FAILED,
-      payload: error
+    .post(`${STUY_SPEC_API_URL}/sections`, section, AXIOS_CONFIG)
+    .then(response => {
+      dispatch({
+        type: t.CREATE_SECTION_SUCCEEDED,
+        payload: response.data
+      });
+      dispatch(fetchSections());
+      dispatch(push("/"));

```

Comment:
As a user, I want to be redirected to the SectionTable after I finish creating a section.